.gitlab-ci/packet.yml

```diff
@@ -151,6 +151,11 @@ packet_rockylinux8-calico:
   extends: .packet_pr
   when: on_success
 
+packet_rockylinux9-calico:
+  stage: deploy-part2
```

Comment:
FYI: To verify this additional job works fine quickly, it is nice to put `unit-tests` here instead of `deploy-part2` temporary.
That change should be another commit in the pull request.
After verifying that works, we can just remove the temporary commit from the pull request to make it review-ready.

Reply:
Thank you for advise.